metpy/calc/tests/test_thermo.py

```diff
@@ -824,7 +825,23 @@ def test_isentropic_pressure_tmp_out():
     tmp[3, :] = 288.
     tmpk = tmp * units.kelvin
     isentlev = [296.] * units.kelvin
-    isentprs = isentropic_interpolation(isentlev, lev, tmpk, tmpk_out=True)
+    isentprs = isentropic_interpolation(isentlev, lev, tmpk, temperature_out=True)
+    truetmp = 296. * units.kelvin
+    assert_almost_equal(isentprs[1], truetmp, 3)
+
+
+def test_isentropic_pressure_tmpk_out_deprecation():
```

Comment:
If this goes in after #1106, you can just use the `@check_and_silence_deprecation` and not even care about the `pytest.warns`.